prow/cmd/pipeline/controller.go

```diff
@@ -352,6 +378,61 @@ func (c *controller) pipelineID(pj prowjobv1.ProwJob) (string, string, error) {
 	return id, url, nil
 }
 
+func createProwJobIdentifier(pj *prowjobv1.ProwJob) string {
+	var additionalIdentifier string
+	if pj.Spec.Refs != nil {
+		var pulls []int
+		for _, pull := range pj.Spec.Refs.Pulls {
+			pulls = append(pulls, pull.Number)
+		}
+		sort.Ints(pulls)
+		additionalIdentifier = fmt.Sprintf("%s/%s@%s %v", pj.Spec.Refs.Org, pj.Spec.Refs.Repo, pj.Spec.Refs.BaseRef, pulls)
+	}
+	return fmt.Sprintf("%s %s", pj.Spec.Job, additionalIdentifier)
```

Comment:
nit: this identifier is only used internally, right? never serialized and never shown to a user. I'm thinking it could be at least loggable: with no pulls or without refs the identifier will have a dangling space...

Reply:
yeah, it is used internally only, but the concept is kind of borrowed from the reconciler from the plank. I think I can remove the space, it was mindless process - the creation of this identifier :)

Reply:
no worries - if this concept is used elsewhere then I'm fine with it